builder/ConfTargets.cmake

```diff
@@ -67,6 +67,7 @@ option( MFX_ENABLE_USER_ENCODE "Enabled user encode plugins?" ON)
 option( MFX_ENABLE_USER_ENC "Enabled user ENC plugins?" ON)
 option( MFX_ENABLE_USER_VPP "Enabled user VPP plugins?" ON)
 
+option( MFX_ENABLE_AV1_VIDEO_DECODE "Enabled AV1 decoder?" OFF)
```

Comment:
I missed that it is OFF by default. Actually, this is too big amount of code which might fail on our compiler checks. Please, make sure you have 2 commits in this PR:
1. This big commit should have `MFX_ENABLE_AV1_VIDEO_DECODE=ON`
2. One small commit on top with `MFX_ENABLE_AV1_VIDEO_DECODE=OFF`

In this way we will be able to verify the build of the big commit under CI.

Reply:
Look relying on API_VERSION_NEXT are better alternative. We get this enabled aoutomagically.